content/en/continuous_delivery/deployments/ciproviders.md

```diff
@@ -0,0 +1,97 @@
+- link: "/continuous_delivery/explorer"
+  tag: "Documentation"
+  text: "Learn how to query and visualize deployment executions"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CD Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLScNhFEUOndGHwBennvUp6-XoA9luTc27XBwtSgXhycBVFM9yA/viewform?usp=sf_link" header="false" >}}
+CD Visibility for CI providers deployments is in private beta. To request access, complete the form.
+{{< /callout >}}
+
+## Overview
+
+Deployments can also be performed within continuous integration pipelines. Typically, these pipelines have a deployment step that is executed after the source code has been tested and the image has been built.
+If you are executing deployments via a CI provider, you can monitor your deployments with CD Visibility if the following requirements are met:
+1. You are using [Pipeline Visibility][1] to monitor your CI pipelines.
+2. Your CI provider supports the "Custom tags and metrics at runtime" feature. You can check this in the [supported features table][2].
+3. You are executing deployments in a CI job (or [related level][9] in your CI provider).
+
+## Setup
+
+<div class="alert alert-info">
+The setup requires the <a href="https://www.npmjs.com/package/@datadog/datadog-ci"> <code>datadog-ci</code> CLI </a> with version `2.26.0` or later.
+</div>
+
+To set up CD Visibility, use the `datadog-ci deployment mark` command inside the CI job that is performing the deployment. Two environment variables are required:
+1. `DD_API_KEY`: Point to your [Datadog API key][4].
+2. `DD_BETA_COMMANDS_ENABLED`: Set to 1.
+
+Optionally, you can point the `DD_SITE` environment variable to a specific [Datadog site][3] (default value is `datadoghq.com`).
+
+You can enrich the deployment event that is generated by the `datadog-ci deployment mark` command using the following parameters:
+- `--env`: the environment to which this deployment is performed. For example, `prod`.
+- `--revision`: the revision/version that is being deployed. For example, `1.0.0` or `v123-456`.
+- `--is-rollback` specifies that the deployment is a rollback.
+- `--tags` is an array of key value pairs in the format `key:value`. These tags are added to the deployment event shown in Datadog.
+
+Use `--no-fail` (default: `false`) to prevent the deployment command from failing if there are issues submitting the data.
+
+Once you have added the command to a CI job, the [**Deployments**][5] and [**Deployment Executions**][6] pages populate with data after pipelines are executed.
+For more information, see [Search and Manage][7] and [CD Visibility Explorer][8].
+
+### Examples
+
+Specify that the CI job executes a deployment to the `prod` environment of version `1.0.0`:
+
+```shell
+export DD_BETA_COMMANDS_ENABLED=1
+export DD_API_KEY="<YOUR_API_KEY>"
+
```

Comment:
do you have a possibility to set the `DD_SITE` automatically depending on which site the customer has selected?

Reply:
We do it in Quality Gates, I have pushed a commit that should do it

Reply:
Confirmed that it works